lib/ansible/plugins/lookup/vars.py

```diff
@@ -83,6 +92,7 @@ def run(self, terms, variables=None, **kwargs):
         default = self.get_option('default')
 
         ret = []
+        terms = self._flatten(terms)
```

Comment:
This seems like it could be a backwards incompatible change.  What if my intention is to pass a list of lists as a single term?

I think we probably just need to document arg unpacking.

Reply:
why i wanted a irc meeting on it, to get  a consensus if we make this the norm for lookkups (it 1/2 is already, but not officially)